docs/reference/migration/migrate_2_0/java.asciidoc

```diff
@@ -78,3 +78,32 @@ WrapperQueryBuilder internally.
 
 The `TermsQueryBuilder#execution` method has been removed as it has no effect, it is ignored by the
  corresponding parser.
+
+==== ImmutableSettings removed
+
+Use now `Settings.builder()` instead of `ImmutableSettings.builder()`.
+
+==== InetSocketTransportAddress removed
+
+Use now `InetSocketTransportAddress(InetSocketAddress address)` instead of `InetSocketTransportAddress(String, int)`.
+You can create an InetSocketAddress instance with `InetSocketAddress(String, int)`. For example:
+`new InetSocketTransportAddress(new InetSocketAddress("127.0.0.1", 0));`.
+
+==== Shading and package relocation removed
+
+Elasticsearch does not shade and relocates packages anymore. So you might need to change your imports and
+use the original library imports.
+
```

Comment:
Elasticsearch used to shade its dependencies and to relocate packages.  We no longer use shading or relocation.  You might need to change your imports to the original library (package?) names: